src/services/copyConfig/rba/policy.test.js

```diff
@@ -42,7 +42,7 @@ describe('User sites test suite', () => {
     const spy = await jest.spyOn(client, 'post')
```

Comment:
You can remove the await from the jest.spyOn(), the spy does not return a Promise so there's no need to have it.